en/docs/learn/design-api/publish-api/publish-to-multiple-external-api-stores.md

```diff
@@ -91,7 +91,7 @@ In this guide, let's use two separate instances of WSO2 API Manager, and let's p
             <!--Configuration to set the Developer Portal URL of the current running APIM deployment. 
             APIs published to external developer portals will be redirected to this URL-->
             
-                <StoreURL>http://<ip_address>:<port>localhost:9463/devportal</StoreURL>
```

Comment:
Are you referring here line no 94 should be port number 9443 ?

Reply:
> @arunpatil1983 Yes correct, I am only stating that line 94 needs to be corrected to 9443

Reply:
@uvindra -Line no 94 has been corrected ,please review and see the file.

Reply:
@uvindra -Line no 94 has been corrected ,please review and see the file.

Reply:
Line no 94 has been corrected ,please review and see the file.

Reply:
Line no 94 has been corrected ,please review and see the file

Reply:
Line no 94 has been corrected ,please review and see the file

Reply:
Line no 94 has been corrected ,please review and see the file